apps/website/docs/react/guides/limitations.md

```diff
@@ -4,6 +4,52 @@ sidebar_position: 2
 
 # Limitations
 
+## Runtime styles
+
+Styles can't be created at runtime which includes dynamic selectors as well.
```

Comment:
This section is missing a part on conditional styles

Reply:
Did not get it, can you please clarify?

Reply:
Got it, will push updates 👍

Reply:
@ling1726 added in 1bbbeae.